spirv_common.hpp

```diff
@@ -107,7 +107,11 @@ inline std::string convert_to_string(float t)
 	// std::to_string for floating point values is broken.
 	// Fallback to something more sane.
 	char buf[64];
+
+	auto old = std::locale::global(std::locale::classic());
```

Comment:
Are you sure std::locale is the right thing to do though? This is a C function, does std::locale affect it?